payment_ippay_ach/controllers/main.py

```diff
@@ -0,0 +1,123 @@
+"""Ippay Cotroller."""
+
+from odoo import http, _
+from odoo.http import request
+from werkzeug.urls import url_encode
+from odoo.addons.payment.controllers.portal import PaymentProcessing
+from odoo.addons.account_payment.controllers.payment import PaymentPortal
+from odoo.addons.portal.controllers.portal import _build_url_w_params
+from odoo.exceptions import ValidationError, UserError
+
+
+error_str = ''
```

Comment:
Remove?

Reply:
`from odoo.exceptions import ValidationError, UserError`
using in ippay_ach_s2s_create_json_3ds() method

Reply:
I was referring to `error_str`